docker/puppetserver-standalone/docker-entrypoint.d/20-use-templates-initially.sh

```diff
@@ -11,3 +11,16 @@ do
     test -f "$TEMPLATES/$f" && cp -np "$TEMPLATES/$f" .
 done
 cd /
+
+# if the data dir is empty, copy over everything from build time
+if [ ! "$(ls -A /opt/puppetlabs/server/data/puppetserver)" ]; then
+  cp -a /var/tmp/puppetserver/* /opt/puppetlabs/server/data/puppetserver/
```

Comment:
FWIW, `cp -a` doesn't seem to work under LCOW in the PR I put up. I had to change to `cp -R` only